client/src/components/UsersListModal/Users.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Button } from '@blueprintjs/core';
+import './Users.css';
+
+class Users extends React.Component {
+  state = {
```

Comment:
you are not modifying state anywhere, due to which it is hard coded, if you are not using it, then please remove it